File: m2/w3/d2/es d2 - tiberi damiano/js/moduli/Persona.js
export class Persona {
    nome;
    cognome;
    eta;
    constructor (_nome, _cognome, _eta) {
        this.nome = _nome;
        this.cognome = _cognome;
        this.eta = _eta;
    }

    confrontaEta(p1, p2) {
        if (p1.eta > p2.eta) {
            return `${p1.nome} ${p1.cognome} è più vecchio di ${p2.nome} ${p2.cognome}`;
        } else if (p1.eta == p2.eta) {
            return ``;
        }
        return `${p2.nome} ${p2.cognome} è più vecchio di ${p1.nome} ${p1.cognome}`
    }
}

File: backend/mese2/week3/day2/EsercizioBellissimo - Damiano e Fratelli Violi/EsercizioSpringMicroServizio/src/main/java/microservizio/model/Persona.java
package microservizio.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Entity
@Table(name = "persone")
@NoArgsConstructor
@AllArgsConstructor
@Data
@ToString
@Builder
public class Persona {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	private String nome;
	private String cognome;
	private Integer eta;

}
